### ISSUE TITLE
Soft Mute Duration

### ISSUE THREAD

**Issue by ghost** [GriefPrevention/GriefPrevention#572]
As a player,
I want a feature for Soft Mute so you can specify how long it will last.
This is so many of us don't have to keep un-muting people every time.

EDIT: Also make it to where the player is notified of the Soft Mute..

**Comment by RoboMWM** [comment]
Duplicate of #572

> Also make it to where the player is notified of the Soft Mute.

This defeats the "soft" in "softmute." Softmute is similar to shadowbanning, and is one of the most effective moderation strategies.

**Comment by RoboMWM** [comment]
> Also make it to where the player is notified of the Soft Mute.

This defeats the "soft" in "softmute." Softmute is similar to shadowbanning, and is one of the most effective moderation strategies.

**Comment by RoboMWM** [comment]
Duplicate of #572

**Comment by RoboMWM** [comment]
> Also make it to where the player is notified of the Soft Mute.

This defeats the "soft" in "softmute." Softmute is similar to shadowbanning, and is one of the most effective moderation strategies.

**Comment by RoboMWM** [comment]
> This is so many of us don't have to keep un-muting people every time.

The intent behind softmute is to effectively be a shadowban. If you find yourself in the position of constantly unmuting, perhaps it would be better to target why such players are getting softmuted in the first place?

**Comment by ghost** [comment]
> > This is so many of us don't have to keep un-muting people every time.
> 
> The intent behind softmute is to effectively be a shadowban. If you find yourself in the position of constantly unmuting, perhaps it would be better to target why such players are getting softmuted in the first place?

But did you forget about the duration part?

EDIT: Of course it's like a "shadowban", but it would be awesome to "shadowtempban".

**Comment by TomLewis** [comment]
Why is this even part of GriefPrevention? Use something for chat, like ChatControl pro.

**Comment by RoboMWM** [comment]
For the same reason PvP and nether portal trap rescues are in here - they're related to preventing or mitigating "grief." There are opinions on both sides regarding whether GP should be broken up or stay as an all-in-one.

**Comment by TomLewis** [comment]
I wish it was entirely focused on claim management and all effort put into the best feature rich claim system with siege (I wish Simpleclans support was added), all the things like Chat management should be handled by a fully fledged chat management system and bans by a punishment system.
Then again, depends on the server size, I guess small servers don't need all the bells and whistles, its hard to make a decision for everyone.
Either way, if Soft mute is staying, it should have a duration, personally I forgot I had it and after about 2-3 years I found out I had 200-300 people softmuted and have to regularly clean the file, I ended up keeping my soft mute banned words to a very minimum and using ChatControl to handle everything else.
